src/utils/mp_gpprof_server/libraries/gamespy/gt2/gt2action/TGAFile.cpp

```diff
@@ -1571,4 +1571,4 @@ extern "C" unsigned char * LoadTGAFile
 {
 	TGAFile tgaFile;
 	return (unsigned char *)tgaFile.LoadTGA(filename, (UINT32 *)width, (UINT32 *)height);
-}
\ No newline at end of file
+}
```

Comment:
Remove the empty line, please. We don't want to produce the diff for no reason. :)